### PR TITLE
77 the ability to proceed to snapshot from inh

### PR DESCRIPTION
**Pull request by rik86future**
**What**: "The ability to proceed to Snapshot
Save resident info. Create a dynamo db entry and redirect to Snapshot."

Part of: https://trello.com/c/RDdaOxJG/77-the-ability-to-proceed-to-snapshot-from-inh-linking-residents-data

**Disclaimer**: I appreciate this is not the final solution, but rather an "_intermediate_" phase to connect other dots

**Overview**
- Change the Create resident form by defaulting values that the user does not care about (TODO: make those value optional in the backend)
- Adds a "NEXT" button to the create resident page so that it redirects to Snapshot when pressed
- Adds a "COMPLETE" page for Snapshot to redirect to
- Changes the "EDIT RESIDENT" to Create or Edit the snapshot (temporary solution for this iteration) so that we can test the user journey.

### CODE REVIEW

File: models/help-requests/help_request.model.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ const axios = require('axios');
 const config = require('../../config');
 const { handleAPIErrors } = require('../../helpers/error');
 
+const SNAPSHOT_URL = process.env.SNAPSHOT_URL
```
Comment (kjdchapman) [inline models/help-requests/help_request.model.js:6]: Does this need adding to the pipeline?

File: public/app.css [hunk 2]
```diff
@@ -4217,6 +4217,9 @@ html.lbh-template > body.lbh-template__body * {
   .js-enabled #address-lookup-fieldset {
     display: block; }
 
+.lbh-form-group label {
```
Comment (kjdchapman) [inline public/app.css:4220]: This looks like it's duplicated in `assets/scss/app.scss`. Does it need to be in both places to work?

File: views/help-requests/help-request-edit.njk [hunk 3]
```diff
@@ -38,6 +38,17 @@
 
     {% if query.Id %}
 
+    <!-- TODO: amend this section later -->
```
Comment (kjdchapman) [inline views/help-requests/help-request-edit.njk:41]: Does leaving this TODO in help? 

I tend to lose track of 'em, and write commit messages to explain my crimes if I know something "should" have been done a different way.

File: models/help-requests/help_request.model.js (hunk 1 above)
Comment (kjdchapman) [reply models/help-requests/help_request.model.js:6]: (or maybe just heroku env vars, I guess?)

Comment (rik86future) [reply models/help-requests/help_request.model.js:6]: Yes Heroku vars, already done

File: public/app.css (hunk 2 above)
Comment (rik86future) [reply public/app.css:4220]: the app.css is automatically generated, I am not sure why we need it in version control, but it was there...

File: views/help-requests/help-request-edit.njk (hunk 3 above)
Comment (rik86future) [reply views/help-requests/help-request-edit.njk]: I have clarified now

File: public/app.css (hunk 2 above)
Comment (kjdchapman) [reply public/app.css:4220]: Sounds like "Let's not worry about this yet"!

**Comment by kjdchapman** [comment]
Thanks for the tweaks/explanations - think this is good to go now?

**Pull request merged**